src/subpackages/core/cellprofiler_core/utilities/hdf5_dict.py

```diff
@@ -1475,12 +1479,12 @@ class HDF5ObjectSet(object):
     DENSE = "dense"
     SPARSE = "sparse"
     ATTR_STALE = "stale"
-    AXIS_LABELS = "label"
-    AXIS_C = "c"
-    AXIS_T = "t"
-    AXIS_Z = "z"
-    AXIS_Y = "y"
-    AXIS_X = "x"
+    AXIS_LABELS = SPARSE_FIELD.label.value
+    AXIS_C = SPARSE_FIELD.c.value
+    AXIS_T = SPARSE_FIELD.t.value
+    AXIS_Z = SPARSE_FIELD.z.value
+    AXIS_Y = SPARSE_FIELD.y.value
+    AXIS_X = SPARSE_FIELD.x.value
```

Comment:
Technically, these are not referenced anywhere in code, but are referenced in some docstrings. Left here for posterity to lower confusion, but really should update the docstrings.